### DISCUSSION TITLE
Modules and subsystems

### DISCUSSION THREAD

**Discussion by gparmeggiani** [structurizr/lite#83] [Q&A]
in the [Visualizing Software Architecture](https://leanpub.com/visualising-software-architecture) book, there's a section about modules and subsystems.

```
If the concept of components and modules, or systems and subsystems, is useful, then feel free to build that into the shared vocabulary that you create.
```

I'm aware of the `terminology` DSL keyword, but how to describe these 2 other layers? Can `groups` be used as a zoom-in element?

**Comment by simonbrowndotje** [comment]
> Can groups be used as a zoom-in element?

Not with Structurizr. You might want to look at https://likec4.dev if you need more levels of zoomable abstraction.
